tests/test_adjoint_jacobian.py

```diff
@@ -214,25 +252,29 @@ def test_Rot_gradient(self, stateprep, theta, dev):
 
         tape.trainable_params = {1, 2, 3}
 
-        calculated_val = dev.adjoint_jacobian(tape)
+        method = dev.compute_derivatives if dev._new_API else dev.adjoint_jacobian
+        calculated_val = method(tape)
 
-        tol = 1e-3 if dev.R_DTYPE == np.float32 else 1e-7
+        if dev._new_API:
+            tol = 1e-3 if dev.c_dtype == np.complex64 else 1e-7
+        else:
+            tol = 1e-3 if dev.R_DTYPE == np.float32 else 1e-7
```

Comment:
We may want to have this block as a function too.

Reply:
Why don't we add the `R_DTYPE` property to `dev`'s class?

Reply:
I proposed in a next question the following: "I think it would be better to simply call the variable 'dtype' and store the complex type. We can also add a 'dtype' attribute to the LightningBase class which will return the complex data type. This way, it will simplify the tests."

Reply:
Are there any old API tests that might use `float64` and `complex64` together or `float32` and `complex128` together? If this is the case then @AmintorDusko 's proposed change could break those tests, though this is easily resolved.

Reply:
I'm not proposing removing anything. I'm suggestion to add an attribute 'dtype' to lightningBase with the complex data type.

Reply:
I'm okay with that change. Definitely will be easier for testing. Do we need input from more people or should I start working on this?

Reply:
You can proceed with the change, and we double-check it in the review. But I don't see any blocker for that.